packages/product/src/loaders/connection.ts

```diff
@@ -62,7 +83,29 @@ async function loadDefault({ database, container }) {
   const entities = Object.values(ProductModels) as unknown as EntitySchema[]
   const orm = await DALUtils.mikroOrmCreateConnection(database, entities)
 
+  return orm.em.fork()
```

Comment:
suggestion: forking is not needed here, it's kinda redundant as forking should take place within the service context

Reply:
if I don't fork, I get the following error from the orm
`Using global EntityManager instance methods for context specific actions is disallowed. If you need to work with the global instance's identity map, use 'allowGlobalContext' configuration option or 'fork()'`

So just for safety reason I won't allow global context and still fork as it happens one time here

Reply:
Interesting. Might be that we're using the global manager somewhere, which should be a bug. We can push this for later.

Reply:
I didn't find it though, in both the integration tests and the module we are forking, lets look at it in a subsequent pr